STYLE_GUIDE.md

```diff
@@ -25,6 +25,12 @@ Parameters can be passed as non-const lvalue reference. [Google rule](https://go
 prohibits semantically valid restriction of not passing null pointer
 and introduces ugly code like `foo(&bar)` or `(*buf)[i]`.
 
+### Test suites naming guide
+
+We use GTest for most of testing code in DALI. TestSuites names should start with a capital letter and end with `Test`. 
+Additionally, both suite and case name mustn't contain underscores (`_`). 
```

Comment:
I don't really agree with that one. Sometimes the underscore is what makes the name of the test readable

Reply:
Have you read the FAQ?